app/src/main/java/pm/gnosis/heimdall/ui/safe/details/transactions/SafeTransactionsFragment.kt

```diff
@@ -49,6 +57,8 @@ class SafeTransactionsFragment : BaseFragment(), ScrollableContainer {
 
     override fun onStart() {
         super.onStart()
+        eventTracker.submit(Event.ScreenView(ScreenId.SAFE_TRANSACTION_LIST))
```

Comment:
This is not correct, as it will be triggered even if the tab is not selected (fragment gets created and initialized in the background).

Either we leave it out for now or you hook it to `getUserVisibilityHint` https://stackoverflow.com/questions/9323279/how-to-test-if-a-fragment-view-is-visible-to-the-user